src/components/icon/icon.spec.js

```diff
@@ -419,6 +419,29 @@ describe('mdIcon service', function() {
         $scope.$digest();
       });
 
+      describe('and the URL is a data URL', function() {
+        var svgData = '<svg><g><circle r="50" cx="100" cy="100"></circle></g></svg>';
+
+        describe('and the data is base64 encoded', function() {
+          it('should return correct SVG markup', function() {
+            var data = 'data:image/svg+xml;base64,' + btoa(svgData);
```

Comment:
In `demoLoadSvgIconsFromUrl/script.js` you have a function `$scope.getAndroidEncoded()`, I do not see a unit test for that type of solution.